keyboards/keyten/diablo/info.json

```diff
@@ -0,0 +1,94 @@
+{
+    "manufacturer": "keyten",
+    "keyboard_name": "Diablo",
+    "maintainer": "key10iq",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "usb": {
+        "vid": "0xEB69",
+        "pid": "0x6003",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["D4", "D1", "F7", "F6", "F5"],
+        "cols": ["B7", "F0", "F1", "F4", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D5", "D3", "D2"]
+    },
+	"community_layouts": ["60_tsangan_hhkb"],
+    "layouts": {
```

Comment:
looking at image provided in respective `readme.md` there appears to be support for more than one physical layout; image shows compatible `60_ansi_wkl` Community Layout.

Based on said image and current layout data, could the following Community Layouts (`layouts/default/`) be added?:
- 60_ansi_tsangan
- 60_wkl
- 60_wkl_split_bs_rshift
- 60_hhkb

Reply:
Apparently the layout options available on the PCB are:
1. Backspace: split or 2u
2. Right Shift: split (HHKB-style 1.75u/1u) or 2.75u

![image](https://telegra.ph/file/3dc4aaa19b1838ba3e5d1.jpg)

And providing the WKL versions of the corresponding layouts would make sense, because the only variant of the top case in the GB ended up being WKL.

So the list of layouts should probably be:
- `LAYOUT_60_ansi_wkl`
- `LAYOUT_60_ansi_wkl_split_bs_rshift`
- `LAYOUT_60_tsangan_hhkb` — aliased to `LAYOUT_all`, includes all possible switch locations; buildable on the bare PCB, but won't fit into with the currently existing cases.

There are no community WKL layouts which have only split Backspace or only split right Shift; the only example of a layout like that is `LAYOUT_60_ansi_wkl_split_rshift` in `idb/idb_60`.

Reply:
@key10iq 
Could Community Layout `60_ansi_tsangan` be added? (which is `60_tsangan_hhkb` with 2u Backspace and 2.75u Right Shift)

Reply:
> @key10iq Could Community Layout `60_ansi_tsangan` be added? (which is `60_tsangan_hhkb` with 2u Backspace and 2.75u Right Shift)

i don't think it's worth adding that many layouts at all, so no

Reply:
If the matrix co-ordinates for 2u Backspace and 2.75u Right Shift could be relayed/shared here, I'd be more than happy to code suggest the remaining supported layouts.